Formula/ser2net.rb

```diff
@@ -21,7 +21,7 @@ class Ser2net < Formula
 
     # Pull request submitted upstream as https://github.com/cminyard/gensio/pull/19
     patch do
-      url "https://github.com/cminyard/gensio/pull/19.patch?full_index=1"
```

Comment:
> Pull request URL's are unstable, use the repository URL instead

why PR URL is. not stable? we actually use PR for lots of places.

Reply:
I guess because open PRs are able to be changed (i.e. new commits can be added, which will affect the SHA-256 checksum)

Shouldn't be a problem for merged PRs though since they probably won't change, if it's even possible to change a PR post-merge